specification/eventhub/resource-manager/Microsoft.EventHub/preview/2021-06-01-preview/namespaces-preview.json

```diff
@@ -0,0 +1,1012 @@
+              "description": "Properties of BYOK Encryption description"
+            },
+            "privateEndpointConnections": {
+              "description": "List of private endpoint connections.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/PrivateEndpointConnection"
+              }
+            },
+            "disableLocalAuth": {
+              "type": "boolean",
+              "description": "This property disables SAS authentication for the Event Hubs namespace."
+            }
+          },
+          "description": "Namespace properties supplied for create namespace operation."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/definitions.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Single Namespace item in List or Get Operation"
+    },
+    "Sku": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of this SKU.",
+          "enum": [
+            "Basic",
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "SkuName",
+            "modelAsString": true
+          }
+        },
+        "tier": {
+          "type": "string",
+          "description": "The billing tier of this particular SKU.",
+          "enum": [
+            "Basic",
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "SkuTier",
+            "modelAsString": true
+          }
+        },
+        "capacity": {
+          "format": "int32",
+          "minimum": 0,
+          "type": "integer",
+          "description": "The Event Hubs throughput units for Basic or Standard tiers, where value should be 0 to 20 throughput units. The Event Hubs premium units for Premium tier, where value should be 0 to 10 premium units."
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "SKU parameters supplied to the create namespace operation"
+    },
+    "Identity": {
+      "x-ms-client-flatten": true,
+      "properties": {
+        "principalId": {
+          "type": "string",
+          "description": "ObjectId from the KeyVault",
+          "readOnly": true
+        },
+        "tenantId": {
+          "type": "string",
+          "description": "TenantId from the KeyVault",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "Type of managed service identity.",
+          "enum": [
+            "SystemAssigned",
+            "UserAssigned",
+            "SystemAssigned, UserAssigned",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "ManagedServiceIdentityType",
+            "modelAsString": false
+          }
+        },
+        "userAssignedIdentities": {
+          "type": "object",
```

Comment:
If you are using `item` ref. The type should be array not object.

Reply:
Confirmed that other swaggers are also using item as type object hence not changing it.